Vostok.Metrics.System/Host/HostMetricsCollectorExtensions.cs

```diff
@@ -15,13 +15,13 @@ public static class HostMetricsCollectorExtensions
         /// <para>Enables reporting of system metrics of the host.</para>
         /// <para>Note that provided <see cref="IMetricContext"/> should contain tags sufficient to decouple these metrics from others.</para>
         /// </summary>
-        public static void ReportMetrics(
+        public static IDisposable ReportMetrics(
```

Comment:
Is this okay to change method signature from `void` to `IDisposable` just like that?
I'm not sure anymore that this won't break anything :/

Reply:
Hope so.